pages/auth/redirect.vue

```diff
@@ -41,6 +49,16 @@ export default class RedirectPage extends Vue {
             method: method as string,
           })
           await this.initWallet(connection)
+          if (!this.currentAddress || !this.signer) return
+          const signature = await this.signMessageMemo('authorize', [
+            'profile',
+            'read:nftbook',
+            'write:nftbook',
+            'read:nftcollection',
+            'write:nftcollection',
```

Comment:
Maybe refactor to a constant for easier maintainance

Reply:
[🎨 Add constant for authorization permission](https://github.com/likecoin/app-like-co/pull/474/commits/5c2c13370611651bc56f227ddcdf07466b7ea02a)